spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRemoveOrphanFilesProcedure.java

```diff
@@ -18,8 +18,10 @@
  */
 package org.apache.iceberg.spark.extensions;
 
+import static java.nio.file.Files.createTempDirectory;
```

Comment:
we typically only statically import stuff from AssertJ (and a few other things) but avoid statically importing stuff like these methods, so please use `Files.createTempDirectory` to be in-line with the rest of the codebase

Reply:
@nastra Sure, got it. I statically import the method here because the `java.nio.file.Files`  is conflicting with the same class name like `org.apache.iceberg.Files`. Can I use the full path like `java.nio.file.Files.createTempDirectory(..)`  here?

Reply:
please use `java.nio.file.Files.createTempDirectory` in that case

Reply:
Sure, thank you. Will update all related lines.

Reply:
Updated.